.circleci/config.yml

```diff
@@ -6,38 +6,23 @@ jobs:
     steps:
       - checkout
 
```

Comment:
Why did you remove the cache step?

Reply:
I'm guessing because we went to use `npm ci` which nukes the node_modules directory before anything...

Reply:
Is this much slower than before?  It seems ok to me, just curious.

Reply:
> I'm guessing because we went to use `npm ci` which nukes the node_modules directory before anything...

Yes.  If the dependencies in `package-lock.json` don't match with `package.json`, `npm ci` will exit with error.
I guess it would be slower, but acceptable.